src/main/kotlin/org/rust/ide/typing/paste/RsConvertJsonToStructCopyPasteProcessor.kt

```diff
@@ -221,7 +312,7 @@ fun generateStruct(
 
 private val NON_IDENTIFIER_REGEX: Regex = Regex("[^a-zA-Z_0-9]")
 
-fun normalizeFieldName(field: String): String {
+private fun normalizeFieldName(field: String): String {
```

Comment:
I think if field name has incorrect characters, we shouldn't apply any normalization at all and directly use original field name in serde `rename` attribute:

```json
{
  "f@": 0,
}
```
```rust
#[derive(Serialize, Deserialize)]
struct Struct {
    #[serde(rename = "f@")]
    pub f: i64,
}
```